lib/src/tooltip_widget.dart

```diff
@@ -130,12 +135,10 @@ class _ToolTipWidgetState extends State<ToolTipWidget>
             viewInsets.bottom;
     final hasSpaceInBottom =
         (actualVisibleScreenHeight - bottomPosition) >= height;
-    return widget.tooltipPosition ??
-        (hasSpaceInTop && !hasSpaceInBottom
-            ? TooltipPosition.top
-            : TooltipPosition.bottom);
+    return hasSpaceInBottom ? TooltipPosition.bottom : TooltipPosition.top;
   }
 
+  // ignore: unused_element
```

Comment:
If using this function why not remove it?

Reply:
We have started using this function so no need to remove it.